web/shared/src/typing_status.ts

```diff
@@ -14,13 +14,24 @@ export type Recipient =
     | (StreamTopic & {
           message_type: "stream";
           notification_event_type: "typing";
-      });
+      })
+    | {
+          notification_event_type: "typing_message_edit";
+          id: number;
```

Comment:
`message_id` is probably a better name here, to be more obviously different from the `ids` above, which probably would benefit from a prep commit renaming it to `user_ids`